package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
The point of my previous comment is that there are 2 customers of add ons - the consuming apps and the developers tasked with its maintenance.  Both should be considered and neither sacrificed for the other.